### PR TITLE
Add more info in README

### LINKED ISSUE #14: Add home page README.md
This README should help users to on-board to the driver in an easy way.

### PR DESCRIPTION
**Pull request by bertinatto**
Also, consolidate README under `deploy/kubernetes`.

Fixes #14 #10.

/assign @leakingtapan

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,12 +2,89 @@
 [![Coverage Status](https://coveralls.io/repos/github/kubernetes-sigs/aws-ebs-csi-driver/badge.svg?branch=master)](https://coveralls.io/github/kubernetes-sigs/aws-ebs-csi-driver?branch=master)
 [![Go Report Card](https://goreportcard.com/badge/github.com/kubernetes-sigs/aws-ebs-csi-driver)](https://goreportcard.com/report/github.com/kubernetes-sigs/aws-ebs-csi-driver)
 [![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Fd-nishi%2Faws-ebs-csi-driver.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2Fd-nishi%2Faws-ebs-csi-driver?ref=badge_shield)
-# csi-ebs
```
Comment (leakingtapan) [inline README.md]: Out of legal concern, we would like to add following at top of the README:
```
WARNING: This driver is in ALPHA currently. This means that there may be potentially backwards compatibility breaking changes moving forward. Do NOT use this driver in a production environment in its current state.

WARNING: The ALPHA driver is NOT compatible with Kubernetes versions <1.12.

DISCLAIMER: This is not an officially supported Amazon product
```
/cc d-nishi

**Comment by d-nishi** [comment]
/ LGTM

File: README.md (hunk 1 above)
Comment (bertinatto) [reply README.md]: Done

**Pull request merged**